packages/core-data/src/private-selectors.ts

```diff
@@ -56,7 +56,12 @@ export const getBlockPatternsForPostType = createRegistrySelector(
  */
 export const getEntityRecordsPermissions = createRegistrySelector( ( select ) =>
 	createSelector(
-		( state: State, kind: string, name: string, ids: string[] ) => {
+		(
+			state: State,
+			kind: string,
+			name: string,
+			ids: string | string[]
```

Comment:
Is accepting `string` on `ids` a documentation fix or a change in behavior?

Either way, I would suggest taking advantage of the typechecking here and make normalization a userland responsibility (if needed, we can show a deprecation message if non-ts consumption is passing a string despite the old param type)

Reply:
The implementation of that method accepts both a single string as well as an array or strings, thus the change. Also, TS raised an error in the same file where that argument was passed as a single.

https://github.com/WordPress/gutenberg/blob/5896920781c52b73b256445df12492826f9d2b91/packages/core-data/src/private-selectors.ts#L95

Also, its usage in the codebase is only as a string not an array of string

Reply:
I have reverted the change for now and fixed the type error instead.